src/main/resources/META-INF/plugin.xml

```diff
@@ -611,6 +613,7 @@
     <depends>org.jetbrains.plugins.yaml</depends>
     <depends>de.espend.idea.php.annotation</depends>
 
+    <depends optional="true">com.jetbrains.php.dql</depends>
```

Comment:
I think we should make the DQL plugin mandatory. Both from a usability POV, but also because we dont currently check whether the plugin is installed. WDYT?

```suggestion
    <depends>com.jetbrains.php.dql</depends>
```

Reply:
IntelliJ is smart enough to handle unknown languages (i.e. DQL plugin is not installed), when calling `registrar.startInjecting(language)`. So there is no need to check that explicitly.

I'm not sure if it is a good idea to have this dependency mandatory. I think there are a lot people out there who only use some Symfony components and thus have the Symfony plugin installed, but don't need/use Doctrine/DQL.

Reply:
As far as I can see, "DQL Support" is bundled by default in PhpStorm. So this affects only people with IntelliJ IDEA Ultimate who have installed PHP plugin manually.